src/main/scala/viper/gobra/frontend/info/implementation/typing/StmtTyping.scala

```diff
@@ -55,7 +55,8 @@ trait StmtTyping extends BaseTyping { this: TypeInfoImpl =>
         assignableTo.errors(exprType(right), exprType(left))(n)
 
     case n@PShortVarDecl(rights, lefts, _) =>
-      if (lefts.forall(pointsToData))
+      // TODO: check that at least one of the variables is new
+      if (lefts.forall{ x => !x.isInstanceOf[PIdnUnk] || pointsToData(x) })
```

Comment:
Why is the inequality to `PIdnUnk` added? And why does at least one variable need to be new?

Reply:
1. my goal here was to have `x.isInstanceOf[PIdnUnk] implies pointsToData(x)`, which is equivalent to the expression in parentheses. Nevertheless, now that I think of it, it would make sense to make Wildcard a subtype of `DataEntity` and thus, I will undo the changes in this line
2. That is mentioned in the [specification for short variable declarations](https://golang.org/ref/spec#Short_variable_declarations), specifically 
> a short variable declaration may redeclare variables provided they were originally declared earlier in the same block (or the parameter lists if the block is the function body) with the same type, and at least one of the non-blank variables is new.